src/components/PoolOverview.tsx

```diff
@@ -91,6 +99,24 @@ function PoolOverview({
             <span className="label">{t("currencyReserves")}</span>
             <span>{`$${formattedData.reserve}`}</span>
           </div>
+          {formattedData.utilization && (
+            <div className="volume">
```

Comment:
add the `margin` class to each of these to add margin to the right and make the spacing correct